spring-cloud-alibaba-examples/oss-example/src/main/java/com/alibaba/cloud/examples/OssController.java

```diff
@@ -67,4 +72,21 @@ public String download() {
 		}
 	}
 
+	@GetMapping("/upload2")
+	public String uploadWithOutputStream() {
+		OssStorageResource ossStorageResource = new OssStorageResource(this.ossClient,
```

Comment:
we should get `OuputStream` by remoteFile, then use `OutputStream#write` to upload.

Don't construct OssStorageResource, validate it with "oss://" protocol Resource

Reply:
like this:

```java
try (OutputStream os = ((WritableResource) this.remoteFile).getOutputStream()) {
    os.write(req.getBytes());
}
```